clients/shared/table.go

```diff
@@ -25,6 +25,6 @@ func CreateTable(ctx context.Context, dwh destination.DataWarehouse, tableData *
 	}
 
 	// Update cache with the new columns that we've added.
-	tc.MutateInMemoryColumns(true, constants.Add, tableData.ReadOnlyInMemoryCols().GetColumns()...)
+	tc.MutateInMemoryColumns(false, constants.Add, tableData.ReadOnlyInMemoryCols().GetColumns()...)
```

Comment:
This should have been false to begin with. I'm going to put up another PR to remove this flag.

Reply:
https://github.com/artie-labs/transfer/pull/1041